PWGJE/EMCALJetTasks/AliEmcalJetTask.cxx

```diff
@@ -944,6 +964,43 @@ Bool_t AliEmcalJetTask::IsJetInPhos(Double_t eta, Double_t phi, Double_t r)
   }
   return kFALSE;
 }
+  
+/**
+ * Load the artificial tracking efficiency TF1 from a file into the member fTrackEfficiencyFunction
+ * @param path Path to the file containing the TF1
+ * @param name Name of the TF1
+ */
+void AliEmcalJetTask::LoadTrackEfficiencyFunction(const char* path, const char* name)
+{
+  TString fname(path);
+  if (fname.BeginsWith("alien://")) {
+    TGrid::Connect("alien://");
+  }
+  
+  TFile* file = TFile::Open(path);
+  
+  if (!file || file->IsZombie()) {
+    ::Error("AliEmcalJetTask", "Could not open artificial track efficiency function file");
```

Comment:
I'd suggest using `AliErrorStream() << "Message\n";` and `AliInfoStream() << "Message\n";` for this function. You won't have to specify the class name, and I think it's better practice to use streams

Reply:
Done